diff --git a/node_modules/@jupyterlite/server-extension/lib/index.js b/node_modules/@jupyterlite/server-extension/lib/index.js
index 55b6a5c..6d1d4f0 100644
--- a/node_modules/@jupyterlite/server-extension/lib/index.js
+++ b/node_modules/@jupyterlite/server-extension/lib/index.js
@@ -160,7 +160,7 @@ const serviceWorkerPlugin = {
     id: '@jupyterlite/server-extension:service-worker',
     autoStart: true,
     provides: IServiceWorkerManager,
     activate: (app) => {
-        return new ServiceWorkerManager();
+        return new ServiceWorkerManager({ workerUrl: PageConfig.getOption('serviceWorkerUrl') });
     },
 };
