diff --git a/node_modules/@jupyterlab/services/src/kernelspec/manager.ts b/node_modules/@jupyterlab/services/src/kernelspec/manager.ts
index 93d2325e418621eced689d40b63976dcb14b0224..c95c600f3f33c4ff82b6d75d8de9c0877e5849ad 100644
--- a/node_modules/@jupyterlab/services/src/kernelspec/manager.ts
+++ b/node_modules/@jupyterlab/services/src/kernelspec/manager.ts
@@ -54,7 +54,7 @@ export class KernelSpecManager
   /**
    * The server settings for the manager.
    */
-  readonly serverSettings: ServerConnection.ISettings;
+  declare readonly serverSettings: ServerConnection.ISettings;
 
   /**
    * Test whether the manager is ready.
diff --git a/node_modules/@jupyterlab/services/src/session/manager.ts b/node_modules/@jupyterlab/services/src/session/manager.ts
index d6dc5a0096222a5208f7443eaab2460588395fc6..f2d5b73f569d7ae2ce4ea71131a4000073cbef4a 100644
--- a/node_modules/@jupyterlab/services/src/session/manager.ts
+++ b/node_modules/@jupyterlab/services/src/session/manager.ts
@@ -51,7 +51,7 @@ export class SessionManager extends BaseManager implements Session.IManager {
   /**
    * The server settings for the manager.
    */
-  readonly serverSettings: ServerConnection.ISettings;
+  declare readonly serverSettings: ServerConnection.ISettings;
 
   /**
    * Test whether the manager is ready.
diff --git a/node_modules/@jupyterlab/services/src/terminal/manager.ts b/node_modules/@jupyterlab/services/src/terminal/manager.ts
index 08895323434764d3111cabfa013996a904e3a908..05e09924df18434abc0c67ed8d7602957d69094d 100644
--- a/node_modules/@jupyterlab/services/src/terminal/manager.ts
+++ b/node_modules/@jupyterlab/services/src/terminal/manager.ts
@@ -58,7 +58,7 @@ export class TerminalManager extends BaseManager implements Terminal.IManager {
   /**
    * The server settings of the manager.
    */
-  readonly serverSettings: ServerConnection.ISettings;
+  declare readonly serverSettings: ServerConnection.ISettings;
 
   /**
    * Test whether the manager is ready.
diff --git a/node_modules/@jupyterlab/services/src/user/index.ts b/node_modules/@jupyterlab/services/src/user/index.ts
index deefd8c06f4af78288c050c271b560a5a9981b9e..9c00f248132d3c9356700ca54c45e953719d2210 100644
--- a/node_modules/@jupyterlab/services/src/user/index.ts
+++ b/node_modules/@jupyterlab/services/src/user/index.ts
@@ -86,7 +86,7 @@ export class UserManager extends BaseManager implements User.IManager {
   /**
    * The server settings for the manager.
    */
-  readonly serverSettings: ServerConnection.ISettings;
+  declare readonly serverSettings: ServerConnection.ISettings;
 
   /**
    * Test whether the manager is ready.
